tests/lmeval/test_lmeval.py

```diff
@@ -152,46 +193,124 @@ def _run_lm_eval(self):
             batch_size=self.lmeval.batch_size,
         )
 
+        # Always use recovery testing
+        self._validate_recovery(results)
+
+        # If absolute metrics provided, show warnings (not failures)
+        if self.lmeval.metrics:
+            self._check_absolute_warnings(results)
+
+    def _validate_recovery(self, compressed_results):
+        """Validate using recovery testing - compare against base model."""
+        base_metrics = self.base_results["results"][self.lmeval.task]
+        compressed_metrics = compressed_results["results"][self.lmeval.task]
+        higher_is_better_map = compressed_results["higher_is_better"][self.lmeval.task]
+
+        # Get recovery threshold(s)
+        recovery_threshold = self.lmeval.recovery_threshold
+        is_dict = isinstance(recovery_threshold, dict)
```

Comment:
As a nit, remove the use of this variable and instead put the condition in the if statement.